src/platform/forms/tests/definitions/bankAccount.unit.spec.jsx

```diff
@@ -1,3 +1,4 @@
+// This is not a real change
```

Comment:
@SebVortex it seems that our team is only being pulled in because of these comments as reviewers. Are they needed? They don't seem to provide much context and I am wondering if they are carry over from past PRs